### PR TITLE
feat: remove linux build scripts for electron

### PR DESCRIPTION
**Pull request by aniruddhadave**
*Description of changes:*
- Removed build scripts for non windows os 

*Testing done:*
- Tested build locally
- Tested build on CodeBuild 
- Integration Tests Passed

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/electron/package.json [hunk 1]
```diff
@@ -20,12 +20,8 @@
     "build:typescript": "tsc",
     "license-checker": "npx license-checker --production --exclude MIT,Apache-2.0,Unlicense,BSD-2-Clause,BSD-3-Clause",
     "generate-attribution": "npx generate-attribution",
-    "build:react": "npx run-script-os",
-    "build:react:win32": "build-scripts\\install-react-deps.bat",
-    "build:react:default": "./build-scripts/install-react-deps.sh",
-    "build:csharp": "npx run-script-os",
-    "build:csharp:win32": "build-scripts\\install-csharp.bat",
```
Comment (xueningl-az) [inline packages/electron/package.json:27]: What's the difference between " win32",  "default", and "build:csharp"?

**Review by cslong — CHANGES_REQUESTED** [review]
can we add a command to be able to run the install-csharp.sh specifically? Or remove the script entirely?

File: packages/electron/package.json (hunk 1 above)
Comment (cslong) [reply packages/electron/package.json:27]: @xueningl-az, is from the run-script-os command, more details here but this doesn't run properly on newer versions of npm/node. https://github.com/charlesguse/run-script-os

Comment (aniruddhadave) [reply packages/electron/package.json:27]: "build:csharp", "build:react" will build the csharp and react packages respectively. "win32"/"default" tells which script to run, .bat for windows and .sh for linux

**Review by cslong — APPROVED** [review]
lgtm

**Pull request merged**